tasks/test.py

```diff
@@ -27,9 +32,10 @@ def test(ctx, targets=None, coverage=False, race=False, use_embedded_libs=False,
     Example invokation:
         inv test --targets=./pkg/collector/check,./pkg/aggregator --race
     """
-    
     targets_list = ctx.targets if targets is None else targets.split(',')
-    build_tags = get_build_tags()  # pass all the build flags for tests
+    # pass all the build flags for tests, unless on Windows
+    exclude = ["docker"] if invoke.platform.WINDOWS else []
```

Comment:
nit: shouldn't this logic of excluding `docker` by default live in `get_build_tags` directly for consistency?

Reply:
I didn't add it to `get_build_tags` because it takes an explicit list of what should be included and what should not and a runtime dependent evaluation in the function body would break the deal with the caller. I'll add a wrapper to avoid `if invoke.platform.WINDOWS` boilerplate around.